dfs/upgraderegistry.go

```diff
@@ -49,6 +51,17 @@ func (dfs *DistributedFilesystem) UpgradeRegistry(svcs []service.Service, tenant
 			glog.Warningf("Cannot parse image name %s under service %s (%s)", image, svc.Name, svc.ID)
 			continue
 		}
+		// download image from old registry
+		if registryHost != "" {
+			glog.V(2).Info("Downloading image %s from %s registry", image, registryHost)
```

Comment:
If this is only supposed to happen on an upgrade, then I suggest removing `V(2)` and letting it log everytime because it's an unusual condition.